src/components/Icon/iconRules.ts

```diff
@@ -69,6 +69,11 @@ const iconRules = {
       },
 
       ...((bordered || circular) && getBorderedStyles(circular, v.borderColor, iconColor)),
+
+      ...(disabled && {
+        color: 'grey',
+        opacity: 0.45,
```

Comment:
Opacity is good as it appears disabled, however, it will still take user interaction.  Let's set `pointerEvents: 'none'` to prevent user interaction.

Then, let's leave the color as the default.  This helps it fit in on any background.  Imagine disabling a red icon on a gray background.  It would disappear if made gray.

Reply:
added
```typescript
{
  opacity: 0.45,
  cursor: 'not-allowed',
}
```
as per discussion we had in #14